Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -81,3 +81,42 @@ parameters:
   pool: replicapool
   ...
 ```
+
+### Mirroring
+
+#### Prerequisites
+
+Enable the mirroring in the default rados namespace(CephBlockPool), refer [CephBlockPool documentation](ceph-block-pool-crd.md#mirroring).
+
+Configure the rados namespace CRD with mirroring spec
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: namespace-a
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # The name of the CephBlockPool CR where the namespace is created.
+  blockPoolName: replicapool
+  mirroring:
+    enabled: false
+    mode: image
+    remoteNamespace: "test"
+    snapshotSchedules:
+```
+
+#### Mirroring Spec
+
+- `mirroring`: Sets up mirroring of the rados namespace
+    - `enabled`: whether mirroring is enabled on that rados namespace (default: false)
+    - `mode`: mirroring mode to run, possible values are "pool" or "image" (required). Refer to the [mirroring modes Ceph documentation](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring) for more details
+    - `remoteNamespace`: Name of the rados namespace on the peer cluster where the namespace should get mirrored.
+    - `snapshotSchedules`: schedule(s) snapshot at the **rados namespace** level. One or more schedules are supported.
+        - `interval`: frequency of the snapshots. The interval can be specified in days, hours, or minutes using d, h, m suffix respectively.
+        - `startTime`: optional, determines at what time the snapshot process starts, specified using the ISO 8601 time format.
+
+!!! note
+    Rados namespace mirroring is only supported from Ceph 20.0.0.0 releases.
+    And Disabling the cephblockpool namespace would require to disable mirroring on all the
```

Comment:
Shouldn't this comment be in the CephBlockPool mirroring? Seems like it doesn't need to be in this doc. 
Also, notes should only contain a single concept, and this concept is different from the v20 requirement.